packages/forms/src/elements/helpers.ts

```diff
@@ -16,14 +16,14 @@ export const firstDotPosition = (num: string): number => {
 	return num.indexOf('.');
 };
 
-export const parseToDecimals = (num: string, decimals: number): number => {
+export const parseToDecimals = (num: string, decimals: number): string => {
```

Comment:
Are you sure that you want a method called parseToDecimal to return a string?  Maybe the method just needs a rename to avoid confusion?

Reply:
`parseToDecimals `method renamed to `adaptValueToFormat`